lib/easy.c

```diff
@@ -181,6 +182,13 @@ static CURLcode global_init(long flags, bool memoryfuncs)
   }
 #endif
 
+#if defined(__APPLE__)
+  if(Curl_macos_init()) {
+    DEBUGF(fprintf(stderr, "Error: Curl_macos_init failed\n"));
+    goto fail;
+  }
+#endif
```

Comment:
You can remove the #ifdefs from this to make it easier on the eyes. Non-macOS builds will have this call optimized away since it can never be true then.

Reply:
Makes sense, thanks.